xtable-api/src/main/java/org/apache/xtable/model/storage/TableFormat.java

```diff
@@ -27,8 +27,9 @@ public class TableFormat {
   public static final String HUDI = "HUDI";
   public static final String ICEBERG = "ICEBERG";
   public static final String DELTA = "DELTA";
+  public static final String PARQUET="PARQUET";
```

Comment:
Please run `mvn spotless:apply` to clean up some of the formatting issues in the draft